### DISCUSSION TITLE
Relate a term to an entry?

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [statamic/cms#5834] [General]
Hi

Is it possible to add a class to a taxonomy term based on it's relationship to the current entry?

I've got a sidebar listing taxonomy terms, these are mounted to a collection called products. The screenshot below shows an entry page and the taxonomy sidebar menu.

Automated Machines is related to the term automation & materials handling.

How would I test for that in antlers?

![Screenshot 2022-04-13 at 17 58 27](https://user-images.githubusercontent.com/72254741/163232230-191a4e2e-3b2c-4123-bdaa-b4f16db9b39a.png)
'

The taxonomy name is `group_1`, I can see there is a relationship - but I'm not sure what the antlers syntax would be?
![Screenshot 2022-04-13 at 18 07 02](https://user-images.githubusercontent.com/72254741/163233788-bb496cf2-b59f-4b5d-b9c7-964a488766db.png)

I'm on `resources/views/products/show.antlers.html`

This doesn't work

```
{{ if group_1.parent.slug === slug }}TRUE {{ /if }}
```

```
Statamic 3.3.5 Pro
Laravel 9.5.1
PHP 8.0.8
Parser: runtime
```

**Comment by jasonvarga** [comment]
Try:

```
{{ if page:group_1 | raw | contains(slug) }}
  yup, the current page has this term
{{ /if }}
```

**Reply by stephenmeehanuk** [reply]
Thanks, but I got this error
```
Passed object must have a __toString method
```
Removing `raw` also errors the same way

**Reply by edalzell** [reply]
Might be a bug, try using the regex parser?

`{{ if (page:group_1 raw="true" :contains="slug") }}`

**Reply by stephenmeehanuk** [reply]
Thanks, unfortunately still not working?

Changed `config/statamic/antlers.php` from `'version' => 'runtime'` to `'version' => 'regex'`

Added this to the top of view `views/products/show.antlers.html`, before anything else

```
{{ if (page:group_1 raw="true" :contains="slug") }}
  yup, the current page has this term
{{ /if }}
```

I see this error

```
syntax error, unexpected token "=" on line 1 of: <?php if ((null="true" :contains="slug")): ?> yup, the current page has this term <?php endif; ?>
```
Is there anything else I can try?

**Reply by edalzell** [reply]
sounds like a bug.

**Comment by stephenmeehanuk** [comment]
Can this be moved to issues?

**Reply by edalzell** [reply]
I just tried but it doesn't look like the content transfers over, darn.

Please add a new issue.
